x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -48,7 +26,7 @@ export const registerEnterpriseSearchIntegrations = (
       description: i18n.translate('xpack.enterpriseSearch.integrations.webCrawlerDescription', {
         defaultMessage: 'Add search to your website with the web crawler.',
       }),
-      categories: ['enterprise_search', 'app_search', 'web', 'elastic_stack', 'crawler'],
+      categories: ['app_search', 'web', 'elastic_stack', 'crawler'],
```

Comment:
We're keeping the app_search category?

Reply:
Confirmed that we can remove it, so I've included that in this PR.